dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-6.0/io.openliberty.appSecurity-6.0.feature

```diff
@@ -43,6 +43,7 @@ Subsystem-Name: Application Security 6.0 (Jakarta Security 4.0)
   io.openliberty.security.oidcclientcore.internal.jakarta, \
   io.openliberty.security.jakartasec.4.0.internal, \
   io.openliberty.security.jakartasec.3.0.internal.cdi, \
+  io.openliberty.security.jakartasec.4.0.internal.cdi, \
```

Comment:
Since these changes are being delivered directly to beta, consider if a short update in the beta blog post would be a good idea.